modules/services/images/src/main/java/au/com/shiftyjelly/pocketcasts/images/icons/StarEmpty.kt

```diff
@@ -0,0 +1,60 @@
+package au.com.shiftyjelly.pocketcasts.images.icons
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.PathFillType.Companion.NonZero
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap.Companion.Round
+import androidx.compose.ui.graphics.StrokeJoin
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.ImageVector.Builder
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.images.PocketCastsIcons
+
+val PocketCastsIcons.StarEmpty: ImageVector
```

Comment:
To match how Google does Material icons I thought I would try SVG to Compose. Let me know if you think I should stick to drawable XML files.

Reply:
It looks fine to me.